common/option-model.cpp

```diff
@@ -132,41 +132,39 @@ bool option_model::draw( std::string & error_message,
                                    "algorithm\nClick the button, then draw a rect on the frame" );
         }
 
-#if defined(AUTO_HDR)
-        if (opt == RS2_OPTION_HDR_ENABLED) // TODO: enable this when the auto-hdr feature is fully implemented, currently works via the load/save preset buttons at the top of the viewer - make sure to disable HDR before loading!
+        if( opt == RS2_OPTION_HDR_ENABLED )
         {
-            auto disable_hdr_load = value_as_float() > 0;  // if HDR is enabled, we can't load a new config
+            auto disable_hdr_config = value_as_float() > 0;  // if HDR is enabled, we can't modify config
+            ImGui::SameLine( 0, 10 );
 
-            ImGui::SameLine(0, 10);
-            std::string button_label = "Load HDR";
-            std::string caption = rsutils::string::from() << "Load HDR config from JSON##" << button_label;
-            RsImGui::RsImButton([&]() {
-                if (ImGui::Button(caption.c_str(), { 75, 0 }))
-                {
-                    dev->_options_invalidated = true;
-                    auto ret = file_dialog_open(open_file, "JavaScript Object Notation (JSON | PRESET)\0*.json;*.preset\0", NULL, NULL);
+            std::string button_label = "HDR Config";
+            std::string caption = rsutils::string::from() << "Open HDR Configuration##" << button_label;
```

Comment:
Label is too long. I see only "Open HDR Co" on the button. Make button longer and/or label shorter - "HDR Config"